ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -719,6 +725,19 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+    private func updateSubtitleView(for navigationItem: UINavigationItem?) {
+        guard let navigationItem = navigationItem, let subtitle = navigationItem.subtitle else {
+            // Use the default title view
+            navigationItem?.titleView = nil
+            return
+        }
+
+        let customTitleView = TwoLineTitleView(style: style == .primary ? .light : .dark)
+        customTitleView.setup(title: navigationItem.title ?? "", subtitle: subtitle, interactivePart: .title)
```

Comment:
Shouldn't we exit this function or have an assertionFailure in guard if we don't have a title in a TwoLineTitleView? Is it ever acceptable to only display a subtitle in a TwoLineTitleView?